pkg/apiserver/storage/ram/watch.go

```diff
@@ -25,6 +25,18 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/apiserver/storage"
 )
 
+type bookMarkEvent struct {
```

Comment:
nit: since "bookmark" is one word, I think you should use `bookmark` consistently and not `bookMark`.

Reply:
Done